stdlib/multiprocessing/resource_tracker.pyi

```diff
@@ -0,0 +1,19 @@
+from _typeshed import StrOrBytesPath
+from collections.abc import Sized
+
+__all__ = ["ensure_running", "register", "unregister"]
+
+class ResourceTracker:
+    def __init__(self) -> None: ...
+    def getfd(self) -> int | None: ...
+    def ensure_running(self) -> None: ...
+    def register(self, name: Sized, rtype: object) -> None: ...
+    def unregister(self, name: Sized, rtype: object) -> None: ...
```

Comment:
It's true that these methods can be passed literally any object without raising an exception, so your instinct here to annotate these parameters with `object` was good. However, I feel like it's unlikely that you're actually _meant_ to be passing literally any object to these methods. I think it's more likely that you're meant to be passing some kind of "resource" to these methods (the "shared memory segments, semaphores, etc." referenced in the block comment at the top of the module). I don't know enough about `multiprocessing` to know what the _true_ type is, so I'd rather we went for `_typeshed.Incomplete` for now, which serves as an explicit marker of "we're not quite sure what this type should be".

```suggestion
    def register(self, name: Sized, rtype: Incomplete) -> None: ...
    def unregister(self, name: Sized, rtype: Incomplete) -> None: ...
```

Reply:
That makes sense, updated :) thanks for explaining that 👍